CMakeLists.txt

```diff
@@ -176,6 +185,7 @@ if (ENABLE_PDLL_CONVERSIONS)
   message(STATUS "ENABLE_PDLL_CONVERSIONS")
   target_compile_definitions(vast_settings
     INTERFACE
+      -fexceptions
```

Comment:
add this only in case of `ENABLE_VAST_EXCEPTION` is `ON`

move it to previous `target_compile_definitions`

Reply:
Oops, my mistake. I was intended to add under `ENABLE_VAST_EXCEPTION` but added under different flag. Will fix this.